dask/dataframe/tests/test_dataframe.py

```diff
@@ -4668,3 +4668,62 @@ def test_dot_nan():
 
     assert_eq(s1.dot(s2), dask_s1.dot(dask_s2))
     assert_eq(s2.dot(df), dask_s2.dot(dask_df))
+
+
+# TODO: Can the setup here go into a fixture? or just a private function?
```

Comment:
You should use `pytest.mark.parametrize` to check that various invalid `how` kwargs raise errors.